R/helper_clean_model_data.R

```diff
@@ -21,13 +21,16 @@ clean_model_data <- function(data, datargs, estimator = "") {
   m_formula <- eval_tidy(mfargs[["formula"]])
   m_formula_env <- environment(m_formula)
 
-  args_ignored <- c("fixed_effects", "se_type")
+  args_ignored <- c("fixed_effects", "condition_pr", "se_type")
   # For each ... that would go to model.fram .default, early eval,
   # save to formula env, and point to it
   # subset is also non-standard eval
   to_process <- setdiff(
     names(mfargs),
-    c(setdiff(names(formals(stats::model.frame.default)), "subset"), args_ignored)
+    c(
```

Comment:
this would be clearer as `setdiff(setdiff(...), args_ignored)` - add a small infix helper to utils.R `%setdiff%`

Reply:
of `%s-%` for short.

Reply:
not right now thanks.